survey-runner-application/global_vars.tf

```diff
@@ -55,7 +55,7 @@ variable "dns_zone_name" {
 # EB Configuration
 variable "aws_elastic_beanstalk_solution_stack_name" {
   description = "Elastic Beanstalk Amazon Linux version"
-  default     = "64bit Amazon Linux 2016.03 v2.1.0 running Python 3.4"
+  default     = "64bit Amazon Linux 2016.09 v2.3.1 running Python 3.4"
```

Comment:
This should be changed under a separate commit. It will be confusing in the history otherwise